packages/fhir-converter/src/templates/cda/Sections/Assessments.hbs

```diff
@@ -2,9 +2,9 @@
 {{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.8')}}
     {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.8')}}
         {{#each (toArray 2_16_840_1_113883_10_20_22_2_8) as |assessment|}}
-            {{#if assessment.text}}
-                {{#with (extractAndMapTableData assessment.text) as |assessmentText|}}
-                    {{>References/DiagnosticReport/assessment.hbs diagReport=assessment text=assessmentText ID=(generateUUID (toJsonString assessment))}},
+            {{#if (and assessment.text (contains (toLower assessment.title._) "assessment"))}}
```

Comment:
only do this for sections with titles that include "assessment". Based on an empirical observation about which sections within 2.16.840.1.113883.10.20.22.2.8 have Diagnostic Report quality data